### PR TITLE
Adding seattle childrens config 

### PR DESCRIPTION
**Pull request by ybaeus**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ Currently documentation is available for the following systems:
 - [ROSALIND_UGE](docs/rosalind_uge.md)
 - [SAGE BIONETWORKS](docs/sage.md)
 - [SANGER](docs/sanger.md)
+- [SEATTLECHILRENS](docs/seattlechildrens.md)
```
Comment (CarsonJM) [inline README.md:174]:
```suggestion
- [SEATTLECHILDRENS](docs/seattlechildrens.md)
```

File: conf/seattlechildrens.config [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+//global parameters
+params {
+    config_profile_description = 'The SCRI (seattle childrens research institute) cluster profile'
+    config_profile_contact = 'Research Scientific Computing (@RSC-RP)'
+    config_profile_url = 'https://github.com/RSC-RP'
+
+    // SCRI HPC project params
+    queue                       = "paidq" // freeq
+    project                     = ""      // please put your own project code
```
Comment (CarsonJM) [inline conf/seattlechildrens.config:11]: The config file is not editable by the user, so maybe include a param here

```suggestion
    project                     = "${params.project}" 
```

**Review by CarsonJM — CHANGES_REQUESTED** [review]

**Comment by CarsonJM** [comment]
@nf-core-bot fix linting

**Comment by jfy133** [comment]
@CarsonJM @ybaeus if you merge from master, there should be nf-core bot working again (I hope )

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Comment by jfy133** [comment]
@mashehu doesn't seem to be working :cry:

**Review by ybaeus — COMMENTED** [review]
Finishing the review

**Review by jennylsmith — CHANGES_REQUESTED** [review]
It looks great! I just realized that neerja's edits were not merged into `main`. There was some info about the HPC `project info` command line tool. Could you do a pull request for the branch `commits_nk` on our repo?

File: conf/seattlechildrens.config [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+//global parameters
+params {
+    config_profile_description = 'The SCRI (seattle childrens research institute) cluster profile'
+    config_profile_contact = 'Research Scientific Computing (@RSC-RP)'
+    config_profile_url = 'https://github.com/RSC-RP'
+
+    // SCRI HPC project params
+    queue                       = "paidq" // freeq
+    project                     = "${params.project}"
```
Comment (ybaeus) [inline conf/seattlechildrens.config:11]:
```suggestion
    project                     = "${params.project}"
```

Comment (ybaeus) [reply conf/seattlechildrens.config]: remove trailing whitespace

**Comment by mashehu** [comment]
> @mashehu doesn't seem to be working 😢

It does work, the problem is just that editorconfig errors are not autofixed, so somebody needs to manually fix them (unfortunately I can't push to the remote)

File: docs/seattlechildrens.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  - conda-forge
+dependencies:
+  - python>=3.9
+  - nextflow==23.10.0
+  - nf-core==2.10
+  - graphviz
+```
+
+2. Setting channel priority
+   Make sure that channel priority is set to flexible using the following comments:
```
Comment (ybaeus) [inline docs/seattlechildrens.md:32]:
```suggestion
Make sure that channel priority is set to flexible using the following comments:
```

**Review by ybaeus — COMMENTED** [review]
Could anyone please check line 32 of ./docs/seattlechildrens.md?

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: docs/seattlechildrens.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - conda-forge
+dependencies:
+  - python>=3.9
+  - nextflow==23.10.0
+  - nf-core==2.10
+  - graphviz
+```
+
+2. Setting channel priority
+Make sure that channel priority is set to flexible using the following comments:
```
Comment (mashehu) [inline docs/seattlechildrens.md:32]:
```suggestion

Make sure that channel priority is set to flexible using the following comments:
```

Comment (mashehu) [reply docs/seattlechildrens.md:32]: this should finally fix the linting

**Review by jennylsmith — APPROVED** [review]
Looks awesome, thank you!

**Pull request merged**